### ISSUE TITLE
when Alluxio is down，how read data from

### ISSUE THREAD

**Issue by liuzx8888** [Alluxio/alluxio#17093]
When using Running Presto on Iceberg Tables with Alluxio, if the Alluxio service is down, how to read the Iceberg Tables under the hdfs path? I tried to use hive to modify the metadata and DDL statements
` 'alter table person3 set location "hdfs://hadoop1:9000/alluxio/data/mnt/ramdisk/person3'`,
 but it failed. How can I modify the metadata of the iceberg table when Alluxio is down, so that presto or hive can read the data normally and ensure the normal operation of the business

DDL

```
> CREATE TABLE person3 
> (name string, age int, id int)
> STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler'
> LOCATION 'alluxio://192.168.2.180:19998/mnt/ramdisk/person3';
```

**Describe alternatives you've considered**
when Alluxio is down,presto or hive can read the data normally 

**Urgency**
important

**Additional context**
Ensuring the normal operation of the production system is the first priority

**Comment by Kai-Zhang** [comment]
You can try presto + alluxio sdk. This cannot be achieved with alluxio service.

**Comment by liuzx8888** [comment]
@Kai-Zhang Yes, this probably has nothing to do with Alluxio，but I think this situation should be worth considering, the stability of the system put into production is the first requirement we consider, if the HDFS service is down, Alluxio can make the business run normally ， on the contrary, the HDFS service is not down and Alluxio service is down, how to make the system run normally, should also be considered. In addition, the non-iceberg format of the hive management table can be accessed directly after Alluxio is hung up to modify the path, and iceberg This situation I can't think of a good way
----------------------------------------------------------------------------------------------------------------------------
对，这个可能确实跟alluxio本身无关，但是既然想投入到生产系统里去，稳定性肯定是首要考虑因素。极端的情况假如Alluxio就是短时间无法修复的情况下如何保证业务正常开展，这种担忧肯定也是很多客户的担忧。作为一个整体的解决方案肯定是需要有plan B的情况，也感谢老哥的回答啊

**Comment by jja725** [comment]
@liuzx8888 There enterprise feature for that which is called transparent URI, maybe you can take a look https://github.com/Alluxio/alluxio/pull/16823#issuecomment-1473105398
